make/jtags/ft2232d.jtag

```diff
@@ -55,4 +55,9 @@ erase-sig:
 	openocd -f $(PROGRAMMER_CONFIG) -c "transport select $(PROGRAMMER_TRANSPORT)" -f target/efm32.cfg -c "init" -c "reset halt" -c "mww 0x400e0008 0x01" -c "mww 0x400e0010 0x0FE00000" -c "mww 0x400e000c 0x01" -c "mww 0x400e000c 0x02" -c "reset run" -c "shutdown"
 	ftdiutil -f $(PROGRAMMER_CONFIG) -c "init" -c "ftdi_set_bitmode reset" -c "ftdi_rebind_on_exit"
 
+install-fhead: $(PROGRAM_FHEADER)
```

Comment:
Recommend PROGRAM_FHEADER_IMG or something similar

Reply:
Fixed in commit 1c97ae3e08a7b26ed4601fe9774ebfd9040941fe